src/legacy/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -237,7 +237,7 @@ class VisEditorVisualization extends Component {
 
         {!autoApply &&
         <EuiFlexItem grow={false}>
-          <EuiButton iconType="play" fill size="s" onClick={onCommit} disabled={!dirty}>
+          <EuiButton iconType="play" fill size="s" onClick={onCommit} disabled={!dirty} data-test-subj="applyBtn">
```

Comment:
nip: I see you use different notation for naming
```
applyBtn
tsvb_series_value
```
I thinks we should use one rule for all data-test-subj attributes.